os/arch/arm/src/amebad/amebad_serial.c

```diff
@@ -167,36 +167,28 @@
  * UART2_DEV: KM0 log uart
  * UART3_DEV: KM0 luart
  */
+int txint_enable = 0;
```

Comment:
Is it right to use a global variable for this? It should be part of the rtl8721d_up_dev_s??

Reply:
Thanks for the feedback, let me check on this.